sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -270,6 +270,7 @@ public async Task DetectLanguageBatchWithNullTextTest()
             Assert.AreEqual(exceptionMessage, ex.Message);
         }
 
+        #region Analyze Sentiment
```

Comment:
We should definitely start doing this in Form Recognizer as well.

Reply:
I was thinking on dividing the test into separate classes .... not sure yet. but yeah, regions might help for now